tests/integration/lazy-computed-test.js

```diff
@@ -55,15 +55,20 @@ module('Integration | lazy computed', function(hooks) {
   });
 
   test('function syntax: passes the keys when getting', function(assert) {
+    let key2Alias = alias('key2');
     let { subject } = compute({
-      computed: lazyComputed('key1', alias('key2'), getCallback)
+      computed: lazyComputed('key1', key2Alias, getCallback)
     });
 
-    assert.deepEqual(getCallback.args[0], [
-      getValue,
-      { context: subject, key: 'computed', macro: 'key1' },
-      { context: subject, key: 'computed', macro: alias('key2') }
-    ]);
+    let args = getCallback.args[0];
```

Comment:
deepEqual no longer works?

Reply:
See the commit message in the [second commit](https://github.com/kellyselden/ember-macro-helpers/pull/258/commits/ece8d0d6a310ead3f8d10241eb817396e9179045)